online-shop/src/components/ProductList.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import '../productModeling/productList.css';
+import { IProduct } from '../App.js';
+import { Link} from 'react-router-dom';
+
+
+interface IProps {
+    data: IProduct[];
+}
+
+export class ProductList extends React.Component<IProps>{
+    render() {
+        let products= this.props.data.map(
+            (productt) => 
+                <Link to= {`/products/${productt.id}`}>     
```

Comment:
Always assign keys to objects created in loops (like `.map` calls). If you check your console, you get for sure an error / warning because you omitted the keys.